.github/workflows/tag-publish.yaml

```diff
@@ -104,11 +116,12 @@ jobs:
             "${{ steps.image-name.outputs.tagged }}" \
             "${{ steps.image-name.outputs.name }}:latest"
 
-      - name: Deploy to Terra Dev environment
-        uses: broadinstitute/repository-dispatch@master
-        if: github.event_name != 'pull_request'
-        with:
-          token: ${{ secrets.BROADBOT_TOKEN }}
-          repository: broadinstitute/terra-helmfile
-          event-type: update-service
-          client-payload: '{"service": "terraui", "version": "${{ steps.tag.outputs.tag }}", "dev_only": false}'
+  report-to-sherlock:
+    uses: broadinstitute/sherlock/.github/workflows/client-report-app-version.yaml@main
+    needs: tag-push-job
+    with:
+      chart-name: "terraui"
+      new-version: ${{ needs.tag-push-job.outputs.tag }}
+    permissions:
+      contents: "read"
+      id-token: "write"
```

Comment:
This, on the other hand, does not obviously look to me like equivalent behavior. I don't know how `broadinstitute/repository-dispatch` achieved "Deploy to Terra Dev environment" and I don't know what sherlock is.

Reply:
Ah, I think I understand now (now that you've said the same thing a 3rd time 😬). The thing that I was forgetting is that the actual dev deploy is happening in [CircleCI](https://github.com/DataBiosphere/terra-ui/blob/dev/.circleci/config.yml#L255).

Reply:
Exactly, yeah we're still a ways off from deploying to Terra UI's actual environments any other way. Thanks!